docs/docs/technology/awards.md

```diff
@@ -1,33 +1,23 @@
 # Awards and Recognition
 
-In this page, we list the awards and recognition that the technology behind sign.mt has received in the last few years, alongside public independent reviews.
+In this page, we list the awards and recognition that the technology behind sign.mt and Dr. Amit Moryossef has received in the last few years, alongside public independent reviews.
 
-## 2024
-
-- We received an `EMNLP Outstanding Demo Paper Award` for our demonstration of "sign.mt: Real-Time Multilingual Sign Language Translation Application"[^emnlp2024-award].
-
-[^emnlp2024-award]: sign.mt. 2024. [Twitter Post](https://x.com/signmt_/status/1857181686045540787).
-
-- Dr. Amit Moryossef (among others) received the `SwissNLP Award` for his "outstanding contribution to Sign Language Translation in Switzerland"[^swissnlp2024-award].
+Timothy Comment: Not ideal for reader/user, needs to read the award in the top, then scroll down (or click the number for source) then click the source down below again. One click to much. 
 
-[^swissnlp2024-award]: SwissNLP. 2024. [SwissNLP Award](https://swissnlp.org/home/activities/swissnlp-award/).
+## 2024
 
-- sign.mt was independently reviewed for Kazakh-Russian Sign Language translation, reporting accuracy of 37% compared to 98% for human translation[^kazakh-russian-review].
+- We received an `EMNLP Outstanding Demo Paper Award` for our demonstration of "sign.mt: Real-Time Multilingual Sign Language Translation Application". sign.mt. 2024. [Twitter Post](https://x.com/signmt_/status/1857181686045540787).
```

Comment:
to help with the file readability, not website readability, you could create a new line. it doesn't format it as a new line

Reply:
Anyway, a style to write inline citations would be `sign.mt, 2024` (comma, instead of `.`) and then the link, so something like:

```md
- We received an `EMNLP Outstanding Demo Paper Award` for our demonstration of "sign.mt: Real-Time Multilingual Sign Language Translation Application"[^emnlp2024-award]. 
([sign.mt, 2024](https://x.com/signmt_/status/1857181686045540787).

[^emnlp2024-award]: sign.mt. 2024. [Twitter Post](https://x.com/signmt_/status/1857181686045540787).
```

This would add the link in a more prominent, east-to-navigate place, while also including the full citation in references.